src/deploy/function_invitation_contact_ids.sql

```diff
@@ -2,20 +2,32 @@
 -- requires: privilege_execute_revoke
 -- requires: schema_public
 -- requires: table_invitation
+-- requires: table_contact
+-- requires: table_account_block
 -- requires: function_invitation_claim_array
 -- requires: function_events_organized
 -- requires: role_account
 -- requires: role_anonymous
 
 BEGIN;
 
-CREATE FUNCTION maevsi.invitation_contact_ids()
+CREATE OR REPLACE FUNCTION maevsi.invitation_contact_ids()
 RETURNS TABLE (contact_id UUID) AS $$
 BEGIN
   RETURN QUERY
     SELECT invitation.contact_id FROM maevsi.invitation
     WHERE id = ANY (maevsi.invitation_claim_array())
-    OR    event_id IN (SELECT maevsi.events_organized());
+    OR (
+	   event_id IN (SELECT maevsi.events_organized())
+	   AND
```

Comment:
Shouldn't this `AND` be a suffix to the 2 `OR`s? So that the brackets are places around the `OR`, ike `(this OR that) AND not_blocked`?

Reply:
Not necessaril as long as maevsi.invitation_claim_array() considers account blocking in its implementation (as it currently does).

Reply:
As I expect the implementation of `invitation_claim_array` to change, I'd say `(this OR that) AND not_blocked` would ensure that this function won't need to change.